src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -106,14 +106,11 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
 
       <TouchableOpacity onPress={toggleSwitchAnon} style={styles.toggleSwitchAnon}>
         <ToggleSwitch
-          isOn={isAnonymity}
-          onToggle={toggleSwitchAnon}
-          onColor={'#9DEDF1'}
-          circleColor={isAnonymity ? '#00ADB5' : colors.white}
-          label={'Allow anonymous messages? '}
-          offColor="#F5F5F5"
-          size="small"
-          labelStyle={styles.toggleLabel}
+          value={isAnonymity}
+          onValueChange={toggleSwitchAnon}
+          labelLeft="Allow anonymous messages?"
+          circleActiveColor={colors.blue1}
+          activeTextColor={colors.blue1}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The properties passed to the `ToggleSwitch` component have been updated. Make sure these changes do not affect the functionality or appearance of the component in an undesired way.
```diff
-          isOn={isAnonymity}
-          onToggle={toggleSwitchAnon}
-          onColor={'#9DEDF1'}
-          circleColor={isAnonymity ? '#00ADB5' : colors.white}
-          label={'Allow anonymous messages? '}
-          offColor="#F5F5F5"
-          size="small"
-          labelStyle={styles.toggleLabel}
+          value={isAnonymity}
+          onValueChange={toggleSwitchAnon}
+          labelLeft="Allow anonymous messages?"
+          circleActiveColor={colors.blue1}
+          activeTextColor={colors.blue1}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->